packages/sdk/src/types/ContractAddress.ts

```diff
@@ -28,6 +26,34 @@ class ContractAddress implements ContractAddressLike<bigint> {
         /** The subindex of the smart contract address. */
         public readonly subindex: bigint
     ) {}
+
+    /**
+     * Get a string representation of the contract address using the `<index, subindex>` format.
+     * @returns {string} The string representation.
+     */
+    public toString(): string {
+        return `<${this.index}, ${this.subindex}>`;
+    }
+
+    /**
+     * Get a JSON-serializable representation of the contract address.
+     * @returns {ContractAddressLike<bigint>} The JSON-serializable representation.
+     */
+    public toJSON(): ContractAddressLike<bigint> {
```

Comment:
I think it is odd to return something which is not valid JSON from such a method.
I'll take the discussion in the issue related to this.